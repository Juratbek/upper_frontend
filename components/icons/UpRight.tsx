import { FC } from 'react';

import { IIconProps } from './types';

export const UpRightIcon: FC<IIconProps> = () => (
  <svg role='icon' width='20' height='20' viewBox='0 0 20 20' fill='none'>
    <path d='M1.5 2H17.5M17.5 2V18M17.5 2L1.5 18' stroke='#000C1A' strokeWidth='4' />
  </svg>
);
