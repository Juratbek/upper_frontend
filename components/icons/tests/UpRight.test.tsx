import { render, screen } from '@testing-library/react';

import { UpRightIcon } from '../UpRight';

describe('UpRight icon', () => {
  it('renders the icon', () => {
    render(<UpRightIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
