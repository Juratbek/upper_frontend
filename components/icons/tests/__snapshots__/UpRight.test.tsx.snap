// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UpRight icon > renders the icon 1`] = `
<svg
  fill="none"
  height="20"
  role="icon"
  viewBox="0 0 20 20"
  width="20"
>
  <path
    d="M1.5 2H17.5M17.5 2V18M17.5 2L1.5 18"
    stroke="#000C1A"
    stroke-width="4"
  />
</svg>
`;
